scripts/install_systemd_autostart_hero1

```diff
@@ -2,57 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-sudo cp $DIR/ntpdate.service /etc/systemd/system/
-sudo cp $DIR/hero1-roscore.service /etc/systemd/system/
-
-sudo cp $DIR/hero1-battery-conversion.service /etc/systemd/system/
-sudo cp $DIR/hero1-monitor.service /etc/systemd/system/
-sudo cp $DIR/hero1-phone-ui.service /etc/systemd/system/
-sudo cp $DIR/hero1-start.service /etc/systemd/system/
-sudo cp $DIR/hero1-windows-speech.service /etc/systemd/system/
-
-sudo cp $DIR/hero1-vboxautostart-service.service /etc/systemd/system
-sudo cp $DIR/hero1-vboxballoonctrl-service.service /etc/systemd/system
-sudo cp $DIR/hero1-vboxdrv.service /etc/systemd/system
-sudo cp $DIR/hero1-vboxweb-service.service /etc/systemd/system
-
-# Reload and enable
-sudo systemctl daemon-reload
-
-sudo systemctl stop docker.hsrb.robot.service
-sudo systemctl stop docker.hsrb.monitor.service
-sudo systemctl stop docker.hsrb.roscore.service
-
-sudo systemctl disable docker.hsrb.robot.service
-sudo systemctl disable docker.hsrb.monitor.service
-sudo systemctl disable docker.hsrb.roscore.service
-
-sudo systemctl start ntpdate.service
-sudo systemctl start hero1-roscore.service
-
-sudo systemctl start hero1-battery-conversion.service
-sudo systemctl start hero1-monitor.service
-sudo systemctl start hero1-phone-ui.service
-# sudo systemctl start hero1-start.service
-
-sudo systemctl enable ntpdate.service
-sudo systemctl enable hero1-roscore.service
-
-sudo systemctl enable hero1-battery-conversion.service
-sudo systemctl enable hero1-monitor.service
-sudo systemctl enable hero1-phone-ui.service
-# sudo systemctl enable hero1-start.service
-
-sudo systemctl disable vbox*
-sudo systemctl stop vbox*
-
-sudo systemctl enable hero1-vboxautostart-service.service
-sudo systemctl enable hero1-vboxballoonctrl-service.service
-sudo systemctl enable hero1-vboxdrv.service
-sudo systemctl enable hero1-vboxweb-service.service
-sudo systemctl enable hero1-windows-speech.service
-
-sudo systemctl start hero1-windows-speech.service
-
-echo "Check the journal for the log"
-echo "sudo journalctl -fu [job_name, e.g. hero1-monitor, hero1-start]"
+srv_path="$DIR"/services/hero1
+source "$DIR"/install_and_disable_systemd $srv_path hero1-demo
```

Comment:
I would suggest removing all services with the name hero so that we can install from a clean slate.
```suggestion
source "$DIR"/install_and_disable_systemd $srv_path hero
```